packages/commonwealth/client/scripts/views/pages/manage_community/webhooks_form.tsx

```diff
@@ -11,49 +11,47 @@ import { Webhook } from 'models';
 import { notifySuccess, notifyError } from 'controllers/app/notifications';
 import { WebhookSettingsModal } from 'views/modals/webhook_settings_modal';
 import { link, pluralize } from 'helpers';
-import { CWIcon } from '../../components/component_kit/cw_icons/cw_icon';
+import { CWTextInput } from '../../components/component_kit/cw_text_input';
+import { CWButton } from '../../components/component_kit/cw_button';
+import { CWIconButton } from '../../components/component_kit/cw_icon_button';
+import { CWText } from '../../components/component_kit/cw_text';
 
 type WebhooksFormAttrs = {
-  webhooks: Webhook[];
+  webhooks: Array<Webhook>;
 };
 
 export class WebhooksForm implements m.ClassComponent<WebhooksFormAttrs> {
   private disabled: boolean;
   private failure: boolean;
   private success: boolean;
+  private webhookUrl: string;
 
   view(vnode) {
     const { webhooks } = vnode.attrs;
     const chainOrCommObj = { chain: app.activeChainId() };
 
-    const createWebhook = (e) => {
-      e.preventDefault();
-
-      const $webhookInput = $(e.target)
-        .closest('form')
-        .find('[name="webhookUrl"]');
-
-      const webhookUrl = $webhookInput.val();
-
-      if (webhookUrl === null) return;
-
+    const createWebhook = () => {
       this.disabled = true;
       this.success = false;
       this.failure = false;
 
       // TODO: Change to POST /webhook
       $.post(`${app.serverUrl()}/createWebhook`, {
         ...chainOrCommObj,
-        webhookUrl,
+        webhookUrl: this.webhookUrl,
```

Comment:
I know it's sorta out of scope for this PR but we should really have some frontend + backend checks to ensure this is a valid URL. At least worth noting for a todo/ticket.